cannot use library on Angular v13

I start new project with v13 try to serve with import ColorTwitterModule in app.module

and I found this issue

<img width="901" alt="Screen Shot 2564-11-08 at 14 25 57" src="https://user-images.githubusercontent.com/9088059/140700766-d637ac68-b204-4f6b-b5a8-d6253208a83a.png">

environment
- OSX 12.0.1
- Angular 13
- ngx-color 7.3.1

Same issue as others:

```
/mydir/node_modules/ngx-color/fesm2020/ngx-color.mjs:7:0-44 - Error: Module not found: Error: Can't resolve '@ctrl/tinycolor' in '/mydir/node_modules/ngx-color/fesm2020'
```

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng-packagr -p ./src/lib/package.json",
+    "build": "ng-packagr -p ./src/lib/package.json && cp README.md ./dist && cp LICENSE ./dist",
     "ghpages": "ng build --configuration production --no-progress",
     "test": "ng test --watch=false --code-coverage --browsers=ChromeCI",
     "test:watch": "ng test --browsers=ChromeCI",

--- a/src/lib/circle/package.json
+++ b/src/lib/circle/package.json
@@ -6,7 +6,7 @@
     "url": "git+https://github.com/scttcper/ngx-color.git"
   },
   "license": "MIT",
-  "peerDependencies": {
+  "dependencies": {
     "material-colors": "^1.2.6"
   },
   "ngPackage": {

--- a/src/lib/package.json
+++ b/src/lib/package.json
@@ -3,10 +3,12 @@
   "name": "ngx-color",
   "version": "0.0.0-placeholder",
   "description": "A Collection of Color Pickers from Sketch, Photoshop, Chrome & more",
+  "dependencies": {
+    "@ctrl/tinycolor": "^3.4.0"
+  },
   "peerDependencies": {
     "@angular/core": ">=12.0.0-0",
-    "@angular/common": ">=12.0.0-0",
-    "@ctrl/tinycolor": "^3.4.0"
+    "@angular/common": ">=12.0.0-0"
   },
   "homepage": "https://github.com/scttcper/ngx-color",
   "repository": "scttcper/ngx-color",
@@ -15,6 +17,10 @@
     "lib": {
       "entryFile": "public_api.ts"
     },
+    "allowedNonPeerDependencies": [
+      "@ctrl/tinycolor",
+      "material-colors"
+    ],
     "dest": "../../dist"
   },
   "keywords": [

--- a/src/lib/swatches/package.json
+++ b/src/lib/swatches/package.json
@@ -5,7 +5,7 @@
     "type": "git",
     "url": "git+https://github.com/scttcper/ngx-color.git"
   },
-  "peerDependencies": {
+  "dependencies": {
     "material-colors": "^1.2.6"
   },
   "ngPackage": {

